src/zrc/api/tests/test_validation.py

```diff
@@ -1103,6 +1103,7 @@ def test_eigenschap_invalid_resource(self):
 
 class ZaakObjectValidationTests(JWTAuthMixin, APITestCase):
     heeft_alle_autorisaties = True
+    OBJECT = "http://example.org/api/zaakobjecten/8768c581-2817-4fe5-933d-37af92d819dd"
```

Comment:
Global variables wil je inprincipe niet in op klassen niveau definieren. Deze worden meestal op module niveau gedefinieerd ([net zoals je hier doet](https://github.com/VNG-Realisatie/zaken-api/pull/216/files#diff-0f023d3fd72505b8df78422886fb7e8c71d6b770ec615a04c7112c437214ffa0R28)).